src/ts/helpers/hours.ts

```diff
@@ -8,16 +8,20 @@ const subtypes = [
   'off-road',
 ];
 
+export function isHour(offer): boolean {
```

Comment:
please clarify what this does? Does it check if the passed argument is an hour? Or does it check if the offer can be booked by the hour? I believe the latter, but it better be represented in the function name.